support/modules/rootfs-image

```diff
@@ -30,15 +33,28 @@ else
     passive_num=$MENDER_ROOTFS_PART_A_NUMBER
 fi
 passive_num_hex=$(printf '%x' $passive_num)
+upgrade_available="$(${PRINTENV} upgrade_available)"
+upgrade_available="${upgrade_available#upgrade_available=}"
 
 ensure_correct_root_mounted() {
-    if [ "$(stat -L -c %02t%02T $active)" != "$(stat -L -c %04D /)" ]; then
-        echo "Mounted root does not match boot loader environment!" 1>&2
-        exit 1
+    if [ "$(stat -L -c %02t%02T $active)" = "$(stat -L -c %04D /)" ]; then
```

Comment:
Also know you don't agree, but `--dereference`, and `--format`

Reply:
Actually, here I kind of do agree, at least with `--format` which is very unintuitive and uncommon. This is preexisting though, I just extended it.

Reply:
Ah, but here also, busybox doesn't support long options. I think many of the long options are extensions, and POSIX only specifies the short options.

Reply:
Ahh, shite. checkmate.